### PR TITLE
[Doc] support cache select docs for cloud table

### PR DESCRIPTION
**Pull request by starrocks-xupeng**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/data_source/data_cache_warmup.md [hunk 1]
```diff
@@ -15,6 +16,11 @@ Data Cache 预热和 [Data Cache](./data_cache.md) 特性的区别：
 
 该特性从 3.3 版本开始支持。
 
+## 适用场景
```
Comment (Smith-Cruise) [inline docs/zh/data_source/data_cache_warmup.md:19]: 你这个也不算是适用场景吧，怎么感觉是在说使用限制

File: docs/zh/data_source/data_cache_warmup.md [hunk 2]
```diff
@@ -30,20 +36,20 @@ FROM <catalog_name>.<db_name>.<table_name> [WHERE <boolean_expression>]
...
-`CACHE SELECT` 是一个同步过程，且一次只能对一个表进行预热。执行成功后，会返回 Cache 的指标。
+`CACHE SELECT` 是一个同步过程，且一次只能对一个表进行预热。执行成功后，会返回缓存预热相关的指标。
 
 ### 预热远端表所有数据
 
 以下示例拉取外表 `lineitem` 中的所有数据：
 
 ```plaintext
-mysql> cache select * from hive_catalog.test_db.lineitem;
+mysql> cache select * from hive_catalog.test_db.lineitem（如果是存算分离内表则不需要指定 hive_catalog）;
```
Comment (Smith-Cruise) [inline docs/zh/data_source/data_cache_warmup.md:52]: 直接也贴几个存算分离的demo好了

Comment (starrocks-xupeng) [reply docs/zh/data_source/data_cache_warmup.md]: done

File: docs/zh/data_source/data_cache_warmup.md (hunk 1 above)
Comment (starrocks-xupeng) [reply docs/zh/data_source/data_cache_warmup.md:19]: 是适用场景，不是限制，如果他不满足的话也能跑，只是不理想

**Review by Smith-Cruise — APPROVED** [review]

File: docs/zh/data_source/data_cache_warmup.md [hunk 3]
```diff
@@ -15,7 +16,12 @@ Data Cache 预热和 [Data Cache](./data_cache.md) 特性的区别：
 
 该特性从 3.3 版本开始支持。
 
-## 实现方式
+## 适用场景
+1. 缓存盘容量远大于预热的数据量。比如预热要加载 100GB 的数据，如果盘只有 50GB 的话，那么预热加载的后50GB 数据会淘汰前 50GB，只能预热 50GB，达不到预期的预热效果；
+2. 缓存盘数据访问比较稳定。比如预热加载 100GB 的数据，如果盘有200GB，虽然满足了条件1，但如果预热过程中，突然有另一个大冷查询需要加载150GB，或者有大量新数据写入150GB，那么可能会把要预热的数据淘汰，也只能预热50GB，达不到预期的预热效果；
+3. 缓存盘比较独立。比如在存算分离企业版中，新建一个 warehouse，这个 warehouse 内的缓存盘不用作其他数据写入以及查询的用途。
```
Comment (evelynzhaojie) [inline docs/zh/data_source/data_cache_warmup.md:22]: 我们在开源文档里不提企业版的内容

Comment (starrocks-xupeng) [reply docs/zh/data_source/data_cache_warmup.md]: done

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by yingtingdong — APPROVED** [review]

**Pull request merged**